Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Editor/Resources/BMS_Forge_Editor/NetworkManagerTemplate.txt

```diff
@@ -69,6 +69,12 @@ namespace BeardedManStudios.Forge.Networking.Unity
 		[Obsolete("Use Instantiate>:[i]:< instead, its shorter and easier to type out ;)")]
 		public >:[i]:<Behavior Instantiate>:[i]:<NetworkObject(int index = 0, Vector3? position = null, Quaternion? rotation = null, bool sendTransform = true)
 		{
+            if (>:[i]:<NetworkObject.Length <= index)
```

Comment:
Method is obsolete hence not sure what the benefit is to add this here.

Reply:
You're right. That's quite stupid. My intention was to add it to the non-obsolete method, but it seems like I placed it at the wrong place. I added it to the right method in my latest push.